fe1-web/ingestion/handlers/RollCall.ts

```diff
@@ -144,6 +144,12 @@ function handleRollCallCloseMessage(msg: ExtendedMessage): boolean {
     }
   });
 
+  // everyone is automatically subscribed to the reaction channel after the roll call
+  subscribeToChannel(getReactionChannel(lao.id))
+    .catch((err) => {
+      console.error('Could not subscribe to reaction channel, error:', err);
+    });
```

Comment:
Though not strictly needed, I'd suggest putting this and `await`ing it in the `dispatch` call above, so that all async operations are being performed in one go, in a predictable order.